src/pages/Signup/components/FirstSignupForm/FirstSignupForm.Styles.ts

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components"
+
+export const SignupFormLayout = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 50rem;
+  height: 50rem;
+  color: ${({ theme }) => theme.colors.white};
+  gap: 1rem;
+`
+export const SignupFormTitle = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 2rem;
+  width: 100%;
+
+  font-size: ${({ theme }) => theme.fontSizes.large};
+`
+export const SignupFormContainer = styled.form`
+  width: 50rem;
+  height: 48rem;
+`
+
+export const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 4rem;
+
+  position: relative;
+  top: -1.5rem;
+`
+
+export const Button = styled.button<{
+  $width: number
+  $color: string
+  disabled?: boolean
+}>`
+  width: ${({ $width }) => `${$width}%`};
+  height: 100%;
+  padding: 1rem 0;
+  color: ${({ theme }) => theme.colors.white};
+
+  border-radius: ${({ theme }) => theme.radius.size8};
+  font-size: ${({ theme }) => theme.fontSizes.semiLarge};
+  font-weight: ${({ theme }) => theme.fontWeight.bold};
+  transition: all 0.3s ease-in-out;
+
+  box-shadow: 0 0 4px 2px rgba(0, 0, 0, 0.5);
+
+  &:hover {
+    opacity: 0.7;
+    transition: opacity 0.3s ease-in-out;
+  }
+  &:not(:disabled) {
+    background-color: ${({ $color }) => $color};
+  }
```

Comment:
밑에 diabled 조건이 있으니까 이 조건은 없어도 될 것 같아요ㅎㅎ
```suggestion
    background-color: ${({ $color }) => $color};
```

Reply:
현재 취소 버튼과 가입완료 버튼이 모두 같은 styled button 을 써서 이 조건을 지우면 취소 버튼의 색상이 없어지더라고요!
현재는 굳이 나눌 필요성을 못 느껴서 추후에 리팩토링 시에 필요하면 하겠습니다!